templates/deployments.yaml

```diff
@@ -989,6 +989,7 @@ spec:
       tolerations: {{- toYaml . | nindent 8 }}
     {{- end }}
 
+{{- $one_sensor_per_pod := or (gt (len $.Values.st2.packs.sensors) 1) (index $.Values.st2.packs.sensors 0 "ref") }}
```

Comment:
OK, so the sensor mode is automatic/autodetected based on Helm values, right?

Reply:
Yes